go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/jonboulle/clockwork v0.1.0 // indirect
 	github.com/json-iterator/go v1.1.5 // indirect
-	github.com/kubernetes-csi/csi-test v1.1.0
+	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
```

Comment:
Do you know what are the incompatible changes introduced by this csi-test v2.0.0 vs v1.1.0

Reply:
Yes: https://github.com/kubernetes-csi/csi-test/releases/tag/v2.0.0

We should be good.